src/app/tests/suites/certification/Test_TC_ACE_1_1.yaml

```diff
@@ -13,1461 +13,413 @@
+                  },
+              ]
 
     - label:
-          "TH reads the VendorID attribute from the Basic Information cluster
-          (requires view privilege)"
-      verification: |
-          ./chip-tool basic read vendor-id 1 0
-          On TH(Chiptool),Verify  VendorID attribute from the Basic Information cluster
-          288412.817686][3478:3483] CHIP:DMG: {
-          [1660288412.817723][3478:3483] CHIP:DMG:         AttributeReportIBs =
-          [1660288412.817774][3478:3483] CHIP:DMG:         [
-          [1660288412.817818][3478:3483] CHIP:DMG:                 AttributeReportIB =
-          [1660288412.817872][3478:3483] CHIP:DMG:                 {
-          [1660288412.817909][3478:3483] CHIP:DMG:                         AttributeDataIB =
-          [1660288412.817949][3478:3483] CHIP:DMG:                         {
-          [1660288412.818001][3478:3483] CHIP:DMG:                                 DataVersion = 0xe706a702,
-          [1660288412.818042][3478:3483] CHIP:DMG:                                 AttributePathIB =
-          [1660288412.818104][3478:3483] CHIP:DMG:                                 {
-          [1660288412.818152][3478:3483] CHIP:DMG:                                         Endpoint = 0x0,
-          [1660288412.818205][3478:3483] CHIP:DMG:                                         Cluster = 0x28,
-          [1660288412.818266][3478:3483] CHIP:DMG:                                         Attribute = 0x0000_0002,
-          [1660288412.818310][3478:3483] CHIP:DMG:                                 }
-          [1660288412.818356][3478:3483] CHIP:DMG:
-          [1660288412.818412][3478:3483] CHIP:DMG:                                 Data = 65521,
-          [1660288412.818459][3478:3483] CHIP:DMG:                         },
-          [1660288412.818529][3478:3483] CHIP:DMG:
-          [1660288412.818572][3478:3483] CHIP:DMG:                 },
-          [1660288412.818638][3478:3483] CHIP:DMG:
-          [1660288412.818677][3478:3483] CHIP:DMG:         ],
-          [1660288412.818725][3478:3483] CHIP:DMG:
-          [1660288412.818775][3478:3483] CHIP:DMG:         SuppressResponse = true,
-          [1660288412.818818][3478:3483] CHIP:DMG:         InteractionModelRevision = 1
-          [1660288412.818856][3478:3483] CHIP:DMG: }
-          [1660288412.819053][3478:3483] CHIP:TOO: Endpoint: 0 Cluster: 0x0000_0028 Attribute 0x0000_0002 DataVersion: 3875972866
-          [1660288412.819168][3478:3483] CHIP:TOO:   VendorID: 65521
-          [1660288412.819281][3478:3483] CHIP:EM: Sending Standalone Ack for MessageCounter:162210353 on exchange 21063i
-          [1660288412.819381][3478:3483] CHIP:IN: Prepared secure message 0xffff83ffda58 to 0x0000000000000001 (1)  of type 0x10 and protocolId (0, 0) on exchange 21063i with MessageCounter:118354980.
-      disabled: true
+          "Read the NOC attribute (Node operational credentials - requires
```

Comment:
Would be nice if the labels said which test step number it is.

Reply:
added them to this test, but not planning to go add them to all the others :) . If this is something we want, we should document this somewhere for future script writers.